css/main.css

```diff
@@ -29,15 +42,15 @@ a {
   font-weight: 700;
   color: #1f1534;
 }
-.nav ul {
+.nav-ul {
   display: flex;
   margin-bottom: 0;
 }
-.nav ul li {
+.nav-list {
```

Comment:
What is the difference between nav-ul and nav-list? Can you little bit explain, please?